docs/services/data/_index.md

```diff
@@ -152,19 +152,30 @@ err := data.Sync(context.Background(), nil)
 
 ## Use the data management service
 
-To use the data management service, [add the data management service](/services/data/configure-data-capture/#add-the-data-management-service) to your smart machine.
+To use the data management service, [add the data management service](/services/data/configure-data-capture/#add-the-data-management-service) to your machine.
 Then, [configure data capture](/services/data/configure-data-capture/) and [configure cloud sync](/services/data/configure-cloud-sync/) as needed.
 
 For a comprehensive tutorial on using data capture and synchronization together with the ML model service, see [Capture Data and Train a Model](/tutorials/services/data-mlmodel-tutorial/).
 
-### Access, query, and export data
+### Access and export data
```

Comment:
Same comment about access/export via BinaryDataByFilter/TabularDataByFilter using the Filter in order to query in a less flexible way